### ISSUE TITLE
Meetup 22/4: 24 hours longer than 24hours - @nguyentruongky

### ISSUE THREAD

**Issue by TaPhuocHai** [swift-vietnam/meetup#2]
Presentation and source code:
https://github.com/nguyentruongky/Make_the_most_of_24_hours

**Comment by nguyentruongky** [comment]
## Intro
- Manage time to have a good life 
- Manage time to make a boost for your career. 

## Why necessary?
- Practice: Too many developers work too much and make too many bugs. 
- Practice: Many developers can't life and work. 
- Practice: Many developers can't develop themselves. 

## How to solve?

- Life skills
   - Change working time 
   - Keep healthy
   - Change the mindset
- Tech skills 
   - Code base 
   - Auto Layout programmatically 
   - VIP architecture

## Demo with tech skills
- How code base works 
- Why Auto Layout programmatically save time. 
- How fast VIP architecture is

**Comment by TaPhuocHai** [comment]
@nguyentruongky  Do you have something about personal story to tell ? I think It will be interesting about how to you sold the problem.

**Comment by nguyentruongky** [comment]
Sure.

**Comment by TaPhuocHai** [comment]
@nguyentruongky I found something it related with your topic

https://medium.com/product-hunt/these-products-will-help-you-get-more-done-a1f5c5abdaad#.uzefluxgm
https://medium.com/product-hunt/10-mac-menu-bar-apps-you-can-t-live-without-df087d2c6b1#.y335oengm

Do you have a collection of tool to use for increase productivity ?

**Comment by nguyentruongky** [comment]
No, I don't, just share some skills and how to use something exist to manage time and productivity.

**Comment by TaPhuocHai** [comment]
I think it can be great if you have some tool to share with audience.
Maybe It will helpful with another.

You can get tool in above link. 
It is related with your topic, right ?

**Comment by nguyentruongky** [comment]
No, these tools are not what I'm using. I just share some code, some libs, not any tools, maybe only iTerm, my love terminal app.
